index.html

```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
         </ul>
       </section>
     </main>
+    <section class="cards-project">
+      <div class="card">
```

Comment:
- [ ] As mentioned earlier by the previous code reviewer, the list of projects should be grouped using a `ul` tag to group them, and `li` per project as it orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility. 
You have used `ul` - `li` to group the list of project technologies(e.g., "JavasScript", "HTML" and "CSS") but not the actual projects. Kindly make this change and submit it for review.